php/general-functions.php

```diff
@@ -42,7 +42,7 @@ function wp_seo_get_key( $query = false ) {
 		$key = 'archive_author';
 	} elseif ( $query->is_category() || $query->is_tag() || $query->is_tax() ) {
 		$taxonomy = get_queried_object()->taxonomy;
-		$key = "archive_{$taxonomy}";
+		$key = "taxonomy_{$taxonomy}";
```

Comment:
As far as I know, there is no "official" method in the plugin for getting data from the option other than to request it by its array key. If so, changing the key outright seems like a backwards-compatibility break. If so, do you think it's worth accounting for?